_episodes/04-building-containers.md

```diff
@@ -27,6 +27,42 @@ and then deploy the container into a production environment for execution (as yo
   <figcaption>'Singularity usage workflow' via <i>Kurtzer GM, Sochat V, Bauer MW (2017) Singularity: Scientific containers for mobility of compute. PLoS ONE 12(5): e0177459. <a href="https://doi.org/10.1371/journal.pone.0177459">https://doi.org/10.1371/journal.pone.0177459</a></i></figcaption>
 </figure>
 
+# Hello World Singularity
+
+The following recipe shows how to build a hello-world container, and run the container on your local computer.
+
+- Step 1: Open a text editor (e.g., nano, vim, or gedit in a graphical environment)
+
+  ```bash
+  nano hello-world.def
+  ```
+
+- Step 2: Include the following script in the `hello-world.def` file to define the environment
+
+  ```bash
```

Comment:
Does that format OK? (because it's not actually `bash` content...)

Reply:
Yeah locally `...` works.